src/nxdoc/web-ui/web-ui-tutorials/how-to-create-and-customize-nuxeo-actions-menu.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 'HOWTO: Create and Customize a Nuxeo actions menu'
+review:
+    comment: ''
+    date: '2020-03-25'
+    status: ok
+toc: true
+details:
+    howto:
+        excerpt: Learn how to create and customize a Nuxeo actions menu.
+        level: Advanced
+        tool: code
+        topics: Web UI
+labels:
+    - lts2017-ok
+    - lts2019-ok
+    - tutorial
+    - nuxeo-elements
+    - nuxeo-ui-elements
+    - polymer
+tree_item_index: 500
+---
+
+{{! excerpt}}
+In this tutorial you will learn how to create and customize a Nuxeo actions menu on your code.
+{{! /excerpt}}
+
+## Create an actions menu:
+We add a nuxeo actions menu element.
+
+```
+<nuxeo-actions-menu>
+</nuxeo-actions-menu>
+```
+
+We add the menu items, which should consist of actions.
+
+You can add an existing action, for example:
+
+```
+<nuxeo-add-to-collection-button document="[[document]]">
+</nuxeo-add-to-collection-button>
+```
+
+Or you can customize your own.
```

Comment:
I guess the idea here is not so much customizing, because you either have to create a new one or update the code to make it work with the actions meu.